alchemiscale/interface/client.py

```diff
@@ -213,6 +213,14 @@ def get_network_transformations(self, network: ScopedKey) -> List[ScopedKey]:
         """List ScopedKeys for Transformations associated with the given AlchemicalNetwork."""
         return self._query_resource(f"/networks/{network}/transformations")
 
+    def get_network_weight(self, network: ScopedKey) -> float:
+        """Get the weight of the TaskHub associated with a given AlchemicalNetwork."""
+        return self._get_resource(f"/networks/{network}/weight")
+
+    def set_network_weight(self, network: ScopedKey, weight: float) -> None:
+        """Set the weight of the TaskHub associated with a given AlchemicalNetwork."""
```

Comment:
Let's expand these docstrings to explain what changing the `weight` does, especially what setting it to 0 does. We also need to note that `weight should be a float between 0.0 and 1.0.

Happy to discuss this point, or to contribute these docstrings to this PR myself if you prefer!

Reply:
I've updated the docstring, but please feel free to change it!

Reply:
Done! Thanks for adding these!